Removes hardcoded references to MAGFest and TWO Staff Shirts from staff schedule email

RobRuana opened the pull request.

kitsuta commented on uber/templates/emails/shifts/schedule.html:
```diff
@@ -5,23 +5,28 @@
 
-{% if attendee.badge_type == c.STAFF_BADGE %}   
-- Please remember to also pick up your TWO {{ c.EVENT_NAME }} Staff Shirts from the MAGFest MERCH Booth when you get to {{ c.EVENT_NAME }}.  As a reminder, there is an expectation that you will wear one of these two shirts while working your scheduled shifts during {{ c.EVENT_NAME }}, (Thurs-Sunday, not during load-in or load-out).  When not on shift, you should wear a different shirt, so you can ...
+{% if attendee.badge_type == c.STAFF_BADGE and (c.SHIRTS_PER_STAFFER > 0 or c.STAFF_ELIGIBLE_FOR_SWAG_SHIRT) %}
```
This is a minor nitpick, but don't forget that as an HTML email, we can space things out a bit. It's kind of a mass of text like this. :'D

RobRuana replied:
This is totally a personal preference thing, so I can go either way on it, but I prefer breaking text up over multiple lines instead of having one really long horizontal line.

kitsuta replied:
Yeah, same! But you've only used single linebreaks here, so the text as a whole is still 'crowded' feeling since it looks like one big paragraph despite having multiple if-conditions and stuff. Totally a personal preference thing though.

RobRuana replied:
Oh snap! You're right!

kitsuta commented on uber/templates/emails/shifts/schedule.html:
```diff
@@ -5,23 +5,28 @@
+{% if attendee.badge_type == c.STAFF_BADGE and (c.SHIRTS_PER_STAFFER > 0 or c.STAFF_ELIGIBLE_FOR_SWAG_SHIRT) %}
+- Please remember to also pick up your {% if c.SHIRTS_PER_STAFFER > 1 %}{{ c.SHIRTS_PER_STAFFER }}{% endif %}
+{{ c.EVENT_NAME }} {% if c.SHIRTS_PER_STAFFER > 0 %}Staff Shirt{{ c.SHIRTS_PER_STAFFER|pluralize }}{% else %}Shirt{% endif %}
+from the {{ c.EVENT_NAME }} MERCH Booth when you get to {{ c.EVENT_NAME }}.  As a reminder, there is an expectation that you will wear
```
MAGFest staff are, AFAIK, not expected to wear their swag shirts on shift. We'll want to hide the end of this paragraph behind a c.SHIRTS_PER_STAFFER check.

kitsuta commented on uber/templates/emails/shifts/schedule.html:
```diff
@@ -5,23 +5,28 @@
+{% if c.SHIRTS_PER_STAFFER > 1 %}one of these shirts{% else %}this shirt{% endif %} while working your scheduled shifts
+during {{ c.EVENT_NAME }}, (Thurs-Sunday, not during load-in or load-out).  When not on shift, you should wear a different
+shirt, so you can enjoy {{ c.EVENT_NAME }} with everyone else.
 {% endif %}
```
We may also want to consider adding text here to accommodate a situation where staff get both a swag shirt AND staff shirts. This isn't true for any of our events right now, but it covers all our bases for the future. I think the if-check would look something like `if c.SHIRTS_PER_STAFFER > 0 and c.STAFF_ELIGIBLE_FOR_SWAG_SHIRT` and the text could say something like "You also qualify for (an) {{ c.EVENT_NAME }} swag shirt(s). This is the same shirt that attendees can buy and is provided as a perk for staffing."

RobRuana commented:
Ok, I added an `if` statement around the "You must wear your shirt" paragraph. Also added the sentence explaining "swag shirt provided as a perk for staffing". It looks like there are a lot more changes than there actually are, because I spaced out the paragraph like you suggested 😄
